lib/block.rb

```diff
@@ -33,20 +33,20 @@ def self.create(sender, receiver, value)
     switch_db_if_test_env
     result = @con.exec 'INSERT INTO blocks (sender, receiver, value, hash, ' \
             "previous_tx) VALUES ('#{sender}', '#{receiver}', '#{value}', " \
-            "'#{Block.get_hash(sender, receiver, Block.prev_tx_hash)}', " \
```

Comment:
Dan is working on stripping the database logic from the block class

Reply:
Ok - I would just merge these for now and then Dan's branch can replace this